### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by aiwalter** [MichaelisTrofficus/hampel_filter#1]
Hi,

great package! Is this maintained? Could you remove the fixed requirements?
```    install_requires=[
        "numpy>=1.19.2",
        "pandas>=1.1.2"
    ],
    extras_requires={
        "dev": [
            "pytest>=6.0.2"
        ]
    },
    python_requires='>=3.7',
```
I guess for the few lines of code, it should be fine without them? Or at least could you lower them? Also the Python version would be great to make at least `>=3.6`

**Comment by MichaelisTrofficus** [comment]
Hi Martin! You are right, I'll fix it. Thanks for the feedback! 👍

**Comment by aiwalter** [comment]
great, could you release it to PyPI also then? 😃 
Thx!

**Comment by aiwalter** [comment]
I have just seen that you also impute the values. Then I cant use it as it is, because I want to separate outlier detection and imputation.

**Comment by MichaelisTrofficus** [comment]
Yeap, when I made this project my main goal was imputation. By I can output outilier indices as well. No prob.

**Comment by MichaelisTrofficus** [comment]
New version available 0.0.5 in Pypi. Now it returns outlier indices or the imputted timeseries depending on the _imputation_ parameter. Also, there are no fixed requirements and it works for python >= 3.5. Thanks for the comments.
